From e33b3fa0b6f111b54648f682102266b077a0c285 Mon Sep 17 00:00:00 2001
From: Xiang Xiao <xiaoxiang@xiaomi.com>
Date: Sun, 6 Nov 2022 00:51:35 +0800
Subject: [PATCH 09/10] Fix warn: declaration of 'vring_rsc' shadows a previous
 local

regressed by commit 03c80a13417f67f94852cf7d5cb160f1bdf8271e
Author: Tammy Leino <tammy_leino@mentor.com>
Date:   Mon Sep 26 08:14:35 2022 -0700

    handle_vdev_rsc must return error if notifyid cannot be assigned

    Updated handle_vdev_rsc to return error if a unique id cannot be assigned
    Signed-off-by: Tammy Leino <tammy_leino@mentor.com>

Signed-off-by: Xiang Xiao <xiaoxiang@xiaomi.com>
---
 lib/remoteproc/rsc_table_parser.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/remoteproc/rsc_table_parser.c open-amp/lib/remoteproc/rsc_table_parser.c
index 80879380..4cccf4f1 100644
--- a/lib/remoteproc/rsc_table_parser.c
+++ open-amp/lib/remoteproc/rsc_table_parser.c
@@ -144,8 +144,6 @@ int handle_vdev_rsc(struct remoteproc *rproc, void *rsc)
 
 	num_vrings = vdev_rsc->num_of_vrings;
 	for (i = 0; i < num_vrings; i++) {
-		struct fw_rsc_vdev_vring *vring_rsc;
-
 		vring_rsc = &vdev_rsc->vring[i];
 		notifyid = vring_rsc->notifyid;
 		notifyid = remoteproc_allocate_id(rproc,
-- 
2.25.1

